UnityProject/Assets/Scripts/Health/Operable.cs

```diff
@@ -0,0 +1,101 @@
+﻿using System.Collections.Generic;
+using UnityEngine;
+
+namespace Health
+{
+    /// <summary>
+    /// This component allows for a living object to receive surgery
+    /// </summary>
+    [RequireComponent(typeof(HealthSystem))]
+    public class Operable : MonoBehaviour, ICheckedInteractable<HandApply>
+    {
+        protected static readonly StandardProgressActionConfig ProgressConfig
+            = new StandardProgressActionConfig(StandardProgressActionType.Restrain);
+    
+        private RegisterTile registerTile = null;
+        //The initial surgery initiation page
+        public NetTabType NetTabType = NetTabType.Surgery;
+        private HealthSystem healthSystem = null;
+    
+        //Contains all the available surgery options on each body part. 
+        [SerializeField]
+        private SurgeryOptions surgeryOptions = null;
+    
+        /// <summary>
+        /// Dictionary with the active surgery operations on a bodypart.
+        /// </summary>
+        private Dictionary<BodyPartType, SurgeryOperation> activeSurgeries = new Dictionary<BodyPartType, SurgeryOperation>();
+
+        public Dictionary<BodyPartType, SurgeryOperation> ActiveSurgeries
+        {
+            get => activeSurgeries;
+        }
+    
+    
+        void OnEnable()
+        {
+            EnsureInit();
+        }
+
+        private void EnsureInit()
```

Comment:
If this is not doing any operation but assigning the variables, you can do this cleanly inside OnEnable